app/models/buffer_update.rb

```diff
@@ -1,11 +1,11 @@
 class BufferUpdate < ApplicationRecord
-  SOCIAL_SERVICE = {
+  SOCIAL_MEDIA = {
     "twitter" => ApplicationConfig["BUFFER_TWITTER_ID"],
     "facebook" => ApplicationConfig["BUFFER_FACEBOOK_ID"],
     "linkedin" => ApplicationConfig["BUFFER_LINKEDIN_ID"]
   }.freeze
 
-  DEFAULT_SOCIAL_SERVICE = "twitter".freeze
+  DEFAULT_SOCIAL_MIDIA = "twitter".freeze
```

Comment:
@brunnohenrique there's a small typo here :-)

Reply:
adjusted with `git push -f` to don't to pollute the commits